src/data_generator/csv_generator.py

```diff
@@ -38,15 +38,15 @@ def generate(self):
         Generate csv data for training. It generates a 2d dataset and writes it to file.
         """
         super().generate()
-        random.seed(10)
+        np.random.seed(10)
         record_label = 0
         for i in range(self.my_rank, int(self.total_files_to_generate), self.comm_size):
             progress(i+1, self.total_files_to_generate, "Generating CSV Data")
             if (self._dimension_stdev>0):
-                dim1, dim2 = [max(int(d), 0) for d in random.normal( self._dimension, self._dimension_stdev, 2)]
+                dim1, dim2 = [max(int(d), 0) for d in np.random.normal( self._dimension, self._dimension_stdev, 2)]
             else:
                 dim1 = dim2 = self._dimension
-            record = random.random(dim1*dim2)
+            record = np.random.randint(255, size=dim1*dim2, dtype=np.uint8)
             records = [record]*self.num_samples
             df = pd.DataFrame(data=records)
```

Comment:
The part where we calculate dim per sample and do a record can be a common function data_generator.py. This way we can make it more modular.